src/display/scrollbars.js

```diff
@@ -27,26 +27,26 @@ export function measureForScrollbars(cm) {
   }
 }
 
-function NativeScrollbars(place, scroll, cm) {
-  this.cm = cm
-  let vert = this.vert = elt("div", [elt("div", null, null, "min-width: 1px")], "CodeMirror-vscrollbar")
-  let horiz = this.horiz = elt("div", [elt("div", null, null, "height: 100%; min-height: 1px")], "CodeMirror-hscrollbar")
-  place(vert); place(horiz)
-
-  on(vert, "scroll", () => {
-    if (vert.clientHeight) scroll(vert.scrollTop, "vertical")
-  })
-  on(horiz, "scroll", () => {
-    if (horiz.clientWidth) scroll(horiz.scrollLeft, "horizontal")
-  })
-
-  this.checkedZeroWidth = false
-  // Need to set a minimum width to see the scrollbar on IE7 (but must not set it on IE8).
-  if (ie && ie_version < 8) this.horiz.style.minHeight = this.vert.style.minWidth = "18px"
-}
+class NativeScrollbars {
+  constructor(place, scroll, cm) {
+    this.cm = cm
+    let vert = this.vert = elt("div", [elt("div", null, null, "min-width: 1px")], "CodeMirror-vscrollbar")
+    let horiz = this.horiz = elt("div", [elt("div", null, null, "height: 100%; min-height: 1px")], "CodeMirror-hscrollbar")
+    place(vert); place(horiz)
+
+    on(vert, "scroll", () => {
+      if (vert.clientHeight) scroll(vert.scrollTop, "vertical")
+    })
+    on(horiz, "scroll", () => {
+      if (horiz.clientWidth) scroll(horiz.scrollLeft, "horizontal")
+    })
+
+    this.checkedZeroWidth = false
+    // Need to set a minimum width to see the scrollbar on IE7 (but must not set it on IE8).
+    if (ie && ie_version < 8) this.horiz.style.minHeight = this.vert.style.minWidth = "18px"
+  }
 
-NativeScrollbars.prototype = copyObj({
```

Comment:
Err, duh, you made it a regular class, so that's not an issue at all. Ignore this comment.